packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -188,7 +188,9 @@ const DropdownList = React.forwardRef<HTMLDivElement, TDropdownList>((props, lis
                     role='list'
                     ref={list_ref}
                 >
-                    <ThemedScrollbars height={list_dimensions[1] || '200px'}>
+                    <ThemedScrollbars
+                        height={list_dimensions[1] && list_dimensions[1] > 200 ? list_dimensions[1] : '200px'}
```

Comment:
In the test link `height` of `ThemedScrollbars` is always 200px (for Vanillas, Turbos and etc), because max value of `list_dimensions[1]` is `158`. I mean it doesn't break ui as it's applied as max-height